### ISSUE TITLE
Keyboard nav in Menu is weird when mouse pointer is also used

### ISSUE THREAD

**Issue by marmarta** [QubesOS/qubes-issues#7847]
### Qubes OS release
r4.1

### Brief summary
- keyboard focus is not visible enough
- if you hover over an app with mouse and then start navigating with keyboard, you get two hovers (oops)
- if you keyboard-selected something, hovering with mouse behaves weirdly

**Comment by andrewdavidwong** [comment]
Guessing this is referring to the desktop app menu in dom0?
